includes/footer.php

```diff
@@ -45,7 +44,7 @@
                             <img src="/img/sponsors/FIRSTWa.jpg" alt="/img/sponsors/FIRSTWa.jpg" title="FIRST Washington"/>
                         </div>
                         <div class="item">
-                            <img src="/img/sponsors/godaddy.jpg" alt="/img/sponsors/godaddy.jpg" title="Go Daddy"/>
+                            <img src="/img/sponsors/godaddy.jpg" alt="/img/sponsors/godaddy.jpg" title="GoDaddy"/>
                         </div>
                         <div class="item">
                             <img src="/img/sponsors/honeywell.jpg" alt="/img/sponsors/honeywell.jpg" title="Honeywell"/>
```

Comment:
Since all the other logos have white backgrounds, it would look better if this one remained consistent.